docs/content.zh/docs/deployment/cli.md

```diff
@@ -125,6 +125,43 @@ Lastly, you can optionally provide what should be the [binary format]({{< ref "d
 
 The path to the savepoint can be used later on to [restart the Flink job](#starting-a-job-from-a-savepoint).
 
+If the state of the job is quite big, the client will get a timeout exception since it should wait for the savepoint finished.
```

Comment:
```suggestion
If the state size of the job is quite big, the client will get a timeout exception since it has to wait for the savepoint finished.
```